lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -465,6 +465,11 @@ function dosomething_user_new_user($form, &$form_state) {
   global $user;
   $account = $user;
 
+  // Adjust language based on location
+  $user_country_code = dosomething_settings_get_geo_country_code();
+  $language = dosomething_global_convert_country_to_language($user_country_code);
+  user_save($account, array('language' => $language));
```

Comment:
sure thing, mind if I ask, are we moving away from array() or is this a functional thing?